decidim-core/app/mailers/decidim/messaging/conversation_mailer.rb

```diff
@@ -10,27 +10,30 @@ def new_conversation(originator, user, conversation)
           from: originator,
           to: user,
           conversation: conversation,
+          message: conversation.messages.first.body,
```

Comment:
Shouldn't we send the last unread message instead of the first one?

Reply:
With an existing conversation, we send an email with the first unread message. Showing the last unread message when a new conversation is created sounds weird to me, and also I cannot ensure this will be the last unread, as this code is executed in a background job when the conversation is created.

Reply:
Oh, I meant the first unread. Does `conversation.messages` only return unread messages?

Reply:
No, it returns all of them. To me, the "first unread" is the oldest unread message.

This line is inside the `new_conversation` method. This is only called when User A starts a conversation with User B, and they hadn't talked before. Imagine creating a chat room. In this case, User A creates the room with a first message, then a job is scheduled to deliver the notification to User B. User A can keep adding new messages, but we only send an email for the first unread message. 

If the conversation already exists, and all users have read all messages, and User A sends a couple of new messages, then we only send an email for the first unread message, keeping the behaviour of the previous example.

Reply:
Ohhhh, now I get it. Thanks.